keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -279,6 +279,7 @@ def compile(
         box_loss=None,
         classification_loss=None,
         loss=None,
+        metrics=None,
```

Comment:
if this is not supported, then no need to expose this to user

Reply:
the difference is that metrics=[] raised an error before, now it doesn't.  Just a rough edge off the API.

Reply:
Then let's check that through `if kwargs['metrics']`